module/core/src/Application/Command/ChannelExportScheduleConsoleCommand.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * Copyright © Ergonode Sp. z o.o. All rights reserved.
+ * See license.txt for license details.
+ */
+
+declare(strict_types = 1);
+
+namespace Ergonode\Core\Application\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Ergonode\Channel\Domain\Command\Schedule\ScheduleCommand;
+use Ergonode\Core\Application\Installer\InstallerProvider;
+
+/**
+ */
+class ChannelExportScheduleConsoleCommand extends Command
+{
+    private const NAME = 'ergonode:install';
+
+    /**
+     * @var InstallerProvider
+     */
+    private InstallerProvider $provider;
+
+    /**
+     * @param InstallerProvider $provider
+     */
+    public function __construct(InstallerProvider $provider)
```

Comment:
Can't be installers just provided? The provider just brings additional abstraction to the table but no pros.